server/oneoff/sync_name.py

```diff
@@ -0,0 +1,148 @@
+import json
+import os
+from app import session_scope
+from app.database.model_item_translation import ModelItemTranslation
+from app.database.model_spell_translation import ModelSpellTranslation
+
+app_root = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+allowed_file_names = ["items", "mounts", "pets", "weapons", "rhineetles", "spells"]
+
+
+def sync_name_for_item(db_session, file_name):
+    with open(
+        os.path.join(app_root, "app/database/data/{}.json".format(file_name)), "r"
+    ) as file:
+        data = json.load(file)
+        name_to_record_map = {}
+
+        for r in data:
+            name_to_record_map[r["name"]["en"]] = r
+
+        while True:
+            old_name = input(
+                "Enter the old name of the record (as it exists in the db) that you would like to update or 'q' to quit: "
+            )
+            if old_name == "q":
+                return
+            new_name = input(
+                "Enter the new name of the record (as it should appear) that you would like to update: "
+            )
+
+            translations = (
+                db_session.query(ModelItemTranslation)
+                .filter(
+                    ModelItemTranslation.locale == "en",
+                    ModelItemTranslation.name == old_name,
+                )
+                .all()
+            )
+
+            if new_name not in name_to_record_map:
+                print("Error: The new name does not exist within the data files.")
+                continue
+            elif len(translations) > 1:
+                print("Error: Multiple records with the old name exist in the db.")
```

Comment:
It would be good if we handled this better, printing out each of the records in a string and letting the user choose which one to update. It's not urgent; if you don't want to do it now, maybe add a comment with a TODO, and a github issue so we can keep track of it?